sched/adaptdl_sched/policy/pollux.py

```diff
@@ -241,9 +241,8 @@ def get_cluster_utilities(self, states):
         """
         num_replicas = np.sum(states, axis=2)
         speedups = self._get_job_speedups(states)
-        cluster_sizes = self._get_cluster_sizes(states)
         # mask (pop_size x num_nodes): indicates which nodes are active.
-        mask = np.arange(len(self._nodes)) < np.expand_dims(cluster_sizes, 1)
+        mask = np.sum(states, axis=1) > 0
```

Comment:
Can you add few test cases to Pollux that cover the issue that was reported? May be add few other combinations?

Reply:
Added a unit test. Confirmed it fails with the previous code and works with the updated code.